src/Pages/components/Header.js

```diff
@@ -20,23 +25,47 @@ export default function Header({
   if (hasHiddenEvent) sortOptions.push('Hidden')
   if (hasOldEvent) sortOptions.push('Old')
 
+  const chosenEventList = [{ name: 'Public events', value: 'Public' }, { name: 'Favorite events', value: 'Favorite' }, { name: 'Private events', value: 'Private' }]
+  const user = useContext(UserContext);
+  console.log('headerUser', user)
   return (
-    <header>
-      <SelectEvents
-        id="filter"
-        name={sortOptions}
-        options={sortOptions}
-        onSelectEventFilter={onSelectEventFilter}
-        value={eventFilter}
-      />
-      <Searchbar
-        handleEventSuggestion={handleEventSuggestion}
-        handleEventSearch={handleEventSearch}
-      />
-      <CategoryFilter
-        onSelectFilter={onSelectFilter}
-        categoryFilter={categoryFilter}
-      />
-    </header>
+    <StyledHeader>
+      <FilterContainer>
+        <SelectEvents
+          id="filter"
+          name={sortOptions}
+          options={sortOptions}
+          onSelectEventFilter={onSelectEventFilter}
+          value={eventFilter}
+        />
+        <Searchbar
+          handleEventSuggestion={handleEventSuggestion}
+          handleEventSearch={handleEventSearch}
+        />
+        <CategoryFilter
+          onSelectFilter={onSelectFilter}
+          categoryFilter={categoryFilter}
+        />
+      </FilterContainer>
+      {   user ? <EventListFilter
+        onSelectEventListFilter={onSelectEventListFilter}
+        id="EventListfilter"
+        name={chosenEventList}
+        options={chosenEventList}
+        value={chosenEventListFilter} /> : ''}
```

Comment:
If the `else` case is returning an empty string, consider using [`&&`](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).